charts/sn-platform/templates/presto/presto-worker-deployment.yaml

```diff
@@ -74,20 +70,18 @@ spec:
           {{- include "presto.worker.image" . | nindent 10 }}
           {{- if .Values.presto.worker.probe.liveness.enabled }}
           livenessProbe:
-            exec:
-              command:
-                - /bin/bash
-                - /presto/health_check.sh
+            httpGet:
+              path: {{ .Values.presto.worker.probe.liveness.path }}
+              port: {{ .Values.presto.worker.ports.http }}
             initialDelaySeconds: {{ .Values.presto.worker.probe.liveness.initialDelaySeconds }}
             periodSeconds: {{ .Values.presto.worker.probe.liveness.periodSeconds }}
             failureThreshold: {{ .Values.presto.worker.probe.liveness.failureThreshold }}
           {{- end }}
           {{- if .Values.presto.worker.probe.readiness.enabled }}
           readinessProbe:
-            exec:
-              command:
-                - /bin/bash
-                - /presto/health_check.sh
```

Comment:
Should we remove the probe script? it will check if this worker is registered in coordinator successfully.

Reply:
OK, let me test again. I found deployment work fails at liveness and readiness check with this script and keep restarting.